metadata.yaml

```diff
@@ -0,0 +1,3 @@
+homepage: 'https://stape.io/'
```

Comment:
do not commit this file before this PR will not merged.
remove this file from this PR